rosbag2_compression/CMakeLists.txt

```diff
@@ -25,6 +25,17 @@ option(DISABLE_SANITIZERS "Disable the use of gcc sanitizers")
 # https://github.com/google/sanitizers/issues/794
 if(NOT ${CMAKE_SIZEOF_VOID_P} EQUAL 8)
   set(DISABLE_SANITIZERS ON)
+elseif(NOT DISABLE_SANITIZERS)
+  include(CheckCXXSourceCompiles)
+  set(OLD_CMAKE_REQUIRED_FLAGS ${CMAKE_REQUIRED_FLAGS})
+  set(CMAKE_REQUIRED_FLAGS "${OLD_CMAKE_REQUIRED_FLAGS} -fsanitize=leak")
+  check_cxx_source_compiles("int main() {}" HAVE_SANITIZERS)
+  set(CMAKE_REQUIRED_FLAGS ${OLD_CMAKE_REQUIRED_FLAGS})
+  if(NOT HAVE_SANITIZERS)
+    set(DISABLE_SANITIZERS ON)
+    message(WARNING "Sanitizers aren't supported by the compiler - disabling")
+  endif()
+
```

Comment:
@zmichaels11 can you elaborate where the sanitizers are used within the compression package?

Reply:
Personally, I don't know the answer to that. I'll check with my team on it.

Reply:
rosbag2_compression doesn't use sanitizers IIRC.

Reply:
AFAIK we did not add ASAN/TSAN to `rosbag2_compression`

Reply:
maybe a copy&paste mistake? I think we are safe to remove that snippet from the package then.